### PR TITLE
Add BLE central to test BLE.setservice

### PR DESCRIPTION
**Pull request by glistening**
It is a BLE central source to verify BLE pheripheral API.
This is simplified from pizza example's central in noble.

How to test:

- Run test_ble_setservices.js using iotjs.
- Run test_ble_setservices_central.js with node after installing noble.
- "Hello BLE Service" should be printed on success.

IoT.js-DCO-1.0-Signed-off-by: Sanggyu Lee sg5.lee@samsung.com

### CODE REVIEW

File: test/run_pass/test_ble_setservices_central.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+var noble = require('../..');
```
Comment (galpeter) [inline test/run_pass/test_ble_setservices_central.js]: Which module will be loaded this way? AFAIK we don't have anything at that level (`../..`)

Comment (glistening) [inline test/run_pass/test_ble_setservices_central.js]: Oh, thank you for comment. It is modified version of pizza example nodejs noble module. 'test_ble_setservices_central.js` should be copied to `{noble_root}/examples/pizza`. (same location to {noble_root}/examples/pizza/central.js). The require ('../..') is for loading noble itself.

**Comment by yichoi** [comment]
https://travis-ci.org/Samsung/iotjs/jobs/244895223
tidy error

**Comment by glistening** [comment]
@yichoi Fixed.
@galpeter I added one step in test guide in commit log

File: test/run_pass/test_ble_setservices_central.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
+/* Copyright 2016-present Samsung Electronics Co., Ltd. and other contributors
```
Comment (yichoi) [inline test/run_pass/test_ble_setservices_central.js]: 2017-present ?

Comment (glistening) [inline test/run_pass/test_ble_setservices_central.js]: @yichoi I think 2016-present is enough. 2016 is the starting year of IoT.js project. I don't put time to update every copyright year. However, If you prefer, I will fix it.

Comment (yichoi) [inline test/run_pass/test_ble_setservices_central.js]: The first number is for when the file is created not when the project started. FYI iot.js itself started 2015.

Comment (glistening) [inline test/run_pass/test_ble_setservices_central.js]: @yichoi Yes, you're right. Starting year is 2015. It was my mistake. I see. You want to keep the first number as the created year. Although I am not agree with that, I will follow.

**Pull request merged**